client/me/billing-history/billing-history-table.jsx

```diff
@@ -1,57 +1,68 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	includes = require( 'lodash/includes' ),
-	without = require( 'lodash/without' ),
-	bindActionCreators = require( 'redux' ).bindActionCreators,
-	connect = require( 'react-redux' ).connect;
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
 
 /**
  * Internal Dependencies
  */
-var TransactionsTable = require( './transactions-table' ),
-	eventRecorder = require( 'me/event-recorder' ),
-	wpcom = require( 'lib/wp' ).undocumented(),
-	successNotice = require( 'state/notices/actions' ).successNotice,
-	errorNotice = require( 'state/notices/actions' ).errorNotice;
+import TransactionsTable from './transactions-table';
+import eventRecorder from 'me/event-recorder';
 import purchasesPaths from 'me/purchases/paths';
+import { isSendingBillingReceiptEmail } from 'state/selectors';
+import { sendBillingReceiptEmail } from 'state/billing-transactions/actions';
 
 const BillingHistoryTable = React.createClass( {
-	displayName: 'BillingHistoryTable',
-
 	mixins: [ eventRecorder ],
 
-	getInitialState: function() {
-		return {
-			receiptsEmailing: []
-		};
+	emailReceipt( receiptId, event ) {
+		event.preventDefault();
+		this.props.sendBillingReceiptEmail( receiptId );
 	},
 
-	emailReceipt: function( receiptId, event ) {
-		event.preventDefault();
+	renderEmailAction( receiptId ) {
+		const { translate } = this.props;
 
-		this.setState( { receiptsEmailing: this.state.receiptsEmailing.concat( receiptId ) } );
+		if ( this.props.sendingBillingReceiptEmail( receiptId ) ) {
+			return translate( 'Emailing Receipt…' );
```

Comment:
@tyxla thanks - the previous string was actually replaced in this PR. If the string fix had happened in place, then a8c18n wouldn't have posted about it. It did, because the string change was part of a refactor. I'll see if we can prevent comments like this too.